packages/firebase_ui_localizations/lib/src/lang/he.dart

```diff
@@ -135,154 +124,141 @@ class HeLocalizations extends FirebaseUILocalizationLabels {
   String get nullLabel => "null";
 
   @override
-  String get numberLabel => "מספר"; //"number";
+  String get numberLabel => "מספר";
 
   @override
-  String get off => "כבוי"; //"Off";
+  String get off => "כבוי";
 
   @override
-  String get on => "פעיל"; //"On";
+  String get on => "פעיל";
 
   @override
-  String get passwordInputLabel => "סיסמה"; //"Password";
+  String get passwordInputLabel => "סיסמה";
 
   @override
-  String get passwordIsRequiredErrorText =>
-      "נדרשת סיסמה"; //"Password is required";
+  String get passwordIsRequiredErrorText => "נדרשת סיסמה";
 
   @override
   String get passwordResetEmailSentText =>
-      """שלחנו לך דוא"ל עם קישור לאיפוס הסיסמה שלך. אנא בדוק את הדוא"ל שלך."""; //"We've sent you an email with a link to reset your password. Please check your emails.";
+      "שלחנו לך דואייל עם קישור לאיפוס הסיסמה שלך. אנא בדוק את הדואייל שלך.";
```

Comment:
Same as previous comments:
```suggestion
      "שלחנו לך דואייל עם קישור לאיפוס הסיסמה שלך. אנא בדוק את הדואייל שלך.";
```
```suggestion
      """שלחנו לך דוא"ל עם קישור לאיפוס הסיסמה שלך. אנא בדוק את הדוא"ל שלך.""";
```